src/resources/postcss/reset.pcss

```diff
@@ -0,0 +1,357 @@
+.tribe-events {
+
+	/* -----------------------------------------------------------------------------
+	*
+	* Global "Resets"
+	*
+	* ----------------------------------------------------------------------------- */
+
+	* {
+		box-sizing: border-box;
+	}
+
+	/* -----------------------------------------------------------------------------
+	*
+	* HTML5 Elements
+	*
+	* ----------------------------------------------------------------------------- */
+
+	article,
+	aside,
+	details,
+	figcaption,
+	figure,
+	footer,
+	header,
+	main,
+	menu,
+	nav,
+	section,
+	summary {
+		display: block;
+	}
+
+	/* -----------------------------------------------------------------------------
+	*
+	* Normalize "Light"
+	*
+	* See normalize source code for clarity on rules
+	* https://github.com/necolas/normalize.css
+	*
+	* ----------------------------------------------------------------------------- */
+
+	html {
```

Comment:
Ya was looking at compiled but these two can go i imagine.

Reply:
hah yeah good catch, i just copied over the reset from sq1. but definitely wouldn't work the way the compiled selectors are set up.